prow/github/types.go

```diff
@@ -821,7 +821,7 @@ type Team struct {
 	Privacy      string         `json:"privacy,omitempty"`
 	Parent       *Team          `json:"parent,omitempty"`         // Only present in responses
 	ParentTeamID *int           `json:"parent_team_id,omitempty"` // Only valid in creates/edits
-	Permission   TeamPermission `json:"permission"`
```

Comment:
This doesn't have any effect unless you make `TeamPermission` a pointer type, else golang will still always serialize it.

If you do that, please make sure that all code that accesses this field has proper nil-checking.